trie-db/src/triedb.rs

```diff
@@ -371,6 +373,17 @@ impl<'a, H: Hasher, C: NodeCodec<H>> TrieDBIterator<'a, H, C> {
 		}
 		result
 	}
+
+	/// Encoded key for storage lookup
+	fn encoded_key(&self) -> ElasticArray36<u8> {
```

Comment:
is this not also encoded somewhere else?

Reply:
This is a method of `TrieDBIterator`, which manages current key as a vector of nibbles. I considered switching to `NibbleVec` instead, that uses compact form and can be trivially converted to `NibbleSlice`. Unfortunately the conversion would only work for even number of nibbles, and it is not trivial to fix that.